bot/commands/utility/diceRoll.js

```diff
@@ -0,0 +1,23 @@
+const source = require('rfr');
+const randomNumber = require('pure-random-number')
+
+
+const { sendMessage } = source('bot/utils/util');
+const logger = source('bot/utils/logger');
+
+function diceRoleCommand(message) {
```

Comment:
Could we change this command to take an argument for the size of the die instead of defaulting to six?

Example
```
!roll 20
🎲 15
```

Reply:
Did I forget to push my changes? I'm certain I added that.....

Reply:
yup this was already done, I think you reviewed an old version of the code.